x-pack/plugins/security_solution/public/timelines/components/timeline/pinned_tab_content/index.tsx

```diff
@@ -229,6 +230,8 @@ export const PinnedTabContentComponent: React.FC<Props> = ({
                   itemsCount: totalCount,
                   itemsPerPage,
                 })}
+                leadingControlColumns={[defaultControlColumn]}
```

Comment:
Consider creating constants (outside of the component's closure) for the arrays passed to the `leadingControlColumns` and `trailingControlColumns` props to ensure the same array references are passed as props.